### DISCUSSION TITLE
Translation for "in Stock" and "Out Of Stock"

### DISCUSSION THREAD

**Discussion by rasurbrand** [jef/streetmerchant#2899] [Help] [answered]
Hey guys,

I would like to translate the messages "In stock" and "Out of stock" into my local language.
Does anyone know the file where this is configured?

![grafik](https://user-images.githubusercontent.com/46095288/147512101-832a4157-eac8-4011-8f80-f058dedc5866.png)

Thank you very much in advance

**Comment by Mr-Tech-13** [comment, marked as answer]
Under /src it will be logger.ts
